app/src/main/java/ar/com/wolox/android/example/model/New.kt

```diff
@@ -1,3 +1,3 @@
 package ar.com.wolox.android.example.model
 
-class New(var id: Int, var createdAt: String, var title: String, var picture: String, var text: String)
\ No newline at end of file
+class New(var id: Int, var createdAt: String, var title: String, var picture: String, var text: String)
```

Comment:
I forgot to say something, about this. Use `data class` and specify every field as `val`